src/odata-to-abstract-sql.ts

```diff
@@ -176,6 +179,58 @@ const containsQueryOption = (opts?: object): boolean => {
 	return false;
 };
 
+const addNestedFieldSelect = (
+	selectNode: SelectNode[1],
+	fromNode: FromNode[1],
+	fieldName: string,
+	fieldNameAlias: string,
+) => {
+	let aliasName: string | undefined;
+	let tableOrSubqueryNode: FromTypeNode[keyof FromTypeNode];
+	if (isAliasNode(fromNode)) {
+		tableOrSubqueryNode = fromNode[1];
+		aliasName = fromNode[2];
+	} else {
+		tableOrSubqueryNode = fromNode;
+		if (!isTableNode(tableOrSubqueryNode)) {
+			throw new Error('');
```

Comment:
Can we get a better error message here, if nothing else having a placeholder that can be searched for is probably helpful

Reply:
Ends up that was a leftover when I was trying to avoid the `if (aliasNode == null)` check.
Atm it's not needed and is already checked a bit lower at:
```
	if (!isSelectQueryNode(tableOrSubqueryNode)) {
		throw new Error(
			`Adding a nested field select to a subquery containing a ${tableOrSubqueryNode[0]} is not supported`,
		);
	}
```